tools/ports/sdl2_image.py

```diff
@@ -5,8 +5,8 @@
 
 import os
 
-TAG = 'version_4'
```

Comment:
Why was the previous version number on a different scale?

Reply:
> Why was the previous version number on a different scale?

The version in the old `emscripten-ports/SDL2_image` we just made up by @kripken I think.  We are now using upstream versions.